packages/playground/vite.config.ts

```diff
@@ -43,19 +50,23 @@ export default defineConfig({
   server: {
     port: 6660,
     proxy: {
-      "/w": {
+      "/w/*": {
```

Comment:
Why is the asterisk required? I think it should be the same/similar to what we do for `/api`

Reply:
I think `/w/` is probably what works best here.